doc/develop/test/twister.rst

```diff
@@ -149,7 +149,23 @@ name:
 type:
   Type of the board or configuration, currently we support 2 types: mcu, qemu
 simulation:
-  Simulator used to simulate the platform, e.g. qemu.
+  Simulator(s) used to simulate the platform, e.g. qemu.
+
+  .. code-block:: yaml
+
+      simulation:
+        - name: qemu
+        - name: armfvp
+          exec: FVP_Some_Platform
+        - name: custom
+          exec: AnotherBinary
+
+  By default, tests will be executed using the first entry in the simulation array. Another
```

Comment:
Do you mind if this is kept for a future improvement?

This extra attribute would require runtime sanity checks to verify its unicity & lookup to find the default rather than picking index 0, therefore adding more complexity to this PR.